src/components/buttons.js

```diff
@@ -1,8 +1,13 @@
-import React from "react";
-function Navigation( {buttonName, disabled} ){
-   function doClick(){
-       console.log(`${buttonName}`)
-   }
-    return (<button disabled={disabled} onClick={()=> doClick()} >{buttonName}</button>);
```

Comment:
onClick functie is bij App.js geplaatst. (lijkt dat je bij componenten alleen variables wil zetten en alle texten, functies etc in App.js)